File: 8vo Semestre/Taller de Ingenieria Web/ParcialTallerWeb/src/Inventario/Models/Producto.java
package Inventario.Models;

public class Producto {
    String nombre;
    String codigo;
    float precio;
    float stock;

    public Producto(String nombre, String codigo, float precio, float stock) {
        this.nombre = nombre;
        this.codigo = codigo;
        this.precio = precio;
        this.stock = stock;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getCodigo() {
        return codigo;
    }

    public void setCodigo(String codigo) {
        this.codigo = codigo;
    }

    public float getPrecio() {
        return precio;
    }

    public void setPrecio(float precio) {
        this.precio = precio;
    }

    public float getStock() {
        return stock;
    }

    public void setStock(float stock) {
        this.stock = stock;
    }

    @Override
    public String toString() {
        return "Producto{" +
                "nombre='" + nombre + '\'' +
                ", codigo='" + codigo + '\'' +
                ", precio=" + precio +
                ", stock=" + stock +
                '}';
    }
}

File: 5to Semestre/Aplicaciones Moviles/MarketPlace/app/src/main/java/com/example/marketplace/models/Producto.kt
package com.example.marketplace.models

class Producto(
    val id: Int,
    val nombre: String,
    val descripcion: String,
    val precio_actual: Double,
    val created_at: String,
    val updated_at: String,
    val categoria: Categoria
){
    override fun toString(): String {
        return "Id: $id, Nombre: $nombre, Descripcion: $descripcion, Precio: $precio_actual, Categoria: ${categoria.nombre}"
    }
}